automation/fluidity/common-mainnet.yml

```diff
@@ -7,3 +7,6 @@ SECRET_FLU_SENTRY_URL: /fluidity/sentryUrl
 SECRET_FLU_FRONTEND_SENTRY_URL: /fluidity/frontend/sentryURL
 SECRET_FLU_NPM_GITHUB_TOKEN: /fluidity/npm/github_token
 SECRET_FLU_MORALIS_TOKEN: /fluidity/moralis/token
+
+ENV_GTM_ID: GTM-W7QJGR2
```

Comment:
Isn't this set and available in mainnet.yml for app/website already?

Reply:
Only for the app unfortunately. I've just altered it so it's in the "root" common file. 
At least, I'm assuming that's how the fluidity cli code works granted there are other ENV entries in there.

Reply:
Only for the app unfortunately. I've just altered it so it's in the "root" common file.
At least, I'm assuming that's how the fluidity cli code works granted there are other ENV entries in there.